modules/xml/src/main/resources/META-INF/mule-xml.xsd

```diff
@@ -514,14 +514,25 @@ The Jaxen filter allows you to route messages based on XPath expressions. The Ja
                     </xsd:element>
                     <xsd:element name="context-property" type="spring:entryType" minOccurs="0" maxOccurs="unbounded">
                         <xsd:annotation>
-                            <xsd:documentation>A property that wil be made available to the transform context. Expression Evaluators can be used to grab these properties from the message at runtime.
+                            <xsd:documentation>A property that will be made available to the transform context. Expression Evaluators can be used to grab these properties from the message at runtime.
                             </xsd:documentation>
                         </xsd:annotation>
                     </xsd:element>
                 </xsd:sequence>
+                <xsd:attribute name="timeBetweenTransformerEvictions" type="mule:substitutableInt">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                          Transformers are pooled for better throughput, since performing an XSL
```

Comment:
From what i understand you want to achieve, you want two things:
1. Pre-population of the pool at startup. That's already discussed.
2. Eviction of unused objects during low-traffic moments. That's not happening even with this change, since the maxIdle is the same as maxActive by default, so even if the eviction thread is running, it will never deallocate objects from the pool. To do this you will have to also change maxIdle. Anyway, since there is a maxActive in place, the pool will not grow indefinitely and the memory consumption will be limited.